utility/push-homebrew.py

```diff
@@ -20,7 +20,7 @@ def get_github_user():
         CliRuntime().project_config, CliRuntime().get_keychain_key()
     )
     github_config = keychain.get_service("github")
-    return github_config.username, github_config.password
+    return github_config.username, (github_config.password or github_config.token)
```

Comment:
This can just be 
```suggestion
    return github_config.username, github_config.token
```

The deprecated password key is silently copied to token when the service is loaded